repos/system_upgrade/el7toel8/actors/sctpconfigread/libraries/sctpdlm.py

```diff
@@ -0,0 +1,64 @@
+#
+# Functions for probing SCTP usage by DLM
+#
+import re
+
+from six.moves.configparser import SafeConfigParser
+
+from leapp.libraries.stdlib import api
+
+
+def check_dlm_cfgfile():
+    """Parse DLM config file"""
+    fname = "/etc/dlm/dlm.conf"
+
+    try:
+        with open(fname, 'r') as fp:
+            cfgs = '[dlm]\n' + fp.read()
+    except (OSError, IOError):
+        return False
+
+    cfg = SafeConfigParser()
+    try:
+        cfg.read_string(cfgs)
+    except:
```

Comment:
too broad exception specification. IIRC it should be sufficient to check for AttributeError